Add changes suggested by @SiminaB (#789)
greenelab/covid19-review: content/07.pathogenesis.md

## Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through analysis of Viral Genomics and Structure

### Abstract

The novel coronavirus SARS-CoV-2, which emerged in late 2019, was one of the major forces shaping the year 2020 as it spread around the world infecting tens of millions of people with coronavirus disease 2019 (COVID-19).
While this viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.
Because the genomic content of coronaviruses, which specifies the virus's structure, is highly conserved, early genomic analysis provided a significant head start in predicting viral pathogenesis.
The pathogenesis of the virus offers insights into symptomatology, transmission, and individual susceptibility.

> Additionally, prior research into interactions between the human immune system and coronaviruses identifies how these viruses can evade the immune system's protective mechanisms.

Reviewer: Think this should be past tense since its referring to past studies

> Additionally, prior research into interactions between the human immune system and coronaviruses identifies how these viruses can evade the immune system's protective mechanisms.

Author: ```suggestion
Additionally, prior research into interactions between the human immune system and coronaviruses has identified how these viruses can evade the immune system's protective mechanisms.
```

We also explore systems-level research into the regulatory and proteomic effects of SARS-CoV-2 infection and the immune response.
Understanding the structure and behavior of the virus serves to contextualize the many facets of the COVID-19 pandemic and can influence efforts to control the virus and treat the disease.

### Importance

COVID-19 involves a number of organ systems and can present with a wide range of symptoms.
Understanding how the virus infects epithelial cells, however, serves to contextualize how these systems connect.

> Similarly, while the modes of viral transmission have been under debate throughout much of 2020 and the beginning of 2021, the available research suggests that these patterns are very similar to those seen with the closely related viruses SARS-CoV-1 and possibly MERS-CoV.

Reviewer: ```suggestion
Similarly, while the modes of viral transmission have been under debate throughout much of 2020 and the beginning of 2021, the available research suggests that these patterns are very similar to those seen in the closely related viruses SARS-CoV-1 and possibly MERS-CoV.
```

Coronaviruses are large viruses that can be identified by their distinctive "crown-like" shape (Figure @fig:genome-structure).
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers (nm) in which peplomers (protruding structures) of two to three spike (S) glycoproteins are anchored, creating the crown [@isbn:978-0781760607; @doi:10/dvvg2h].
These spikes, which are critical to both viral pathogenesis and to the response by the host immune response, have been visualized using cryo-electron microscopy [@doi:10.1126/science.abb2507].
Because they induce the human immune response, they are also the target of many proposed therapeutic agents, including vaccines [cite therapeutics].
Viral pathogenesis is typically broken down into three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
However, in order to draw a more complete picture of pathogenesis, it is also necessary to examine how infection manifests clinically, identify systems-level interactions between the virus and the human body, and consider the possible effects of variation or evolutionary change on pathogenesis and virulence.

> Thus, both biomedicine and genomics are important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.

Reviewer: Maybe super pick, does `virology` seem a little redundant here. I guess I think of virology as the study of viruses and the clinical medicine and genomics are perspectives through which viruses can be studied...?

> Thus, both biomedicine and genomics are important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.

Author: Thanks @ajlee21! I think maybe just "medicine" vs "biology" would be a better comparison? I basically just want it to say "if you want to understand the medicine, you need to understand the biology (from various subfields), and vice versa." It turns out it's really hard to communicate this!

Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 to 32 kilobases in length [@doi:10.1007/978-1-4939-2438-7_1; @doi:10/ggjr43].
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].

> There are three major genomic regions: one containing the replicase gene (RdRp) and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1] (Figure @fig:genome-structure).

Reviewer: As a non-virologist, this sounds accurate to me

![**Structure of SARS-CoV-2 capsid and genome.**

> A) The genomic structure of coronaviruses is highly conserved, including the order and organization of genes such as the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.

Reviewer 2: Thank you @rando2! I will defer to the virologists!

> A) The genomic structure of coronaviruses is highly conserved, including the order and organization of genes such as the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.

Reviewer 3: Should `The nps include` be `The nsp include`?

> B) The physical structure of the coronavirus virion, including the components encoded by conserved genes S, E, M and N.

Author: See edits above

> B) The physical structure of the coronavirus virion, including the components encoded by conserved genes S, E, M and N.

Reviewer: Just to clarify, what do you mean by "determined by" here? Do you mean they used conserved regions of each of these proteins to predict the structure?

> B) The physical structure of the coronavirus virion, including the components encoded by conserved genes S, E, M and N.

Author: So S, M, E, N are all highly conserved within coronaviruses, and those proteins form the characteristic virion shape (is my understanding)

This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, suggesting that the virus preferentially targets cells located in the lower respiratory tract.
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across individuals.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in 20.4% of patients upon presentation, with fever being present in 30.7%, while another study [51] reported cough (79.4%), fever (77.1%), and dyspnea (56.5%) as the as the most common presenting symptoms; both of these studies considered only hospitalized patients.

> The variability of both which symptoms present and their severity has presented challenges for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.

Reviewer 3: Was this the sentence that was duplicated and removed in a recent pull request?  There have been several edits to this section so we'll need to be careful merging.

Reviewer: Yea this is the PR I remember: https://github.com/greenelab/covid19-review/pull/772

Although respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134], SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidneys [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.

### Clinical Presentation of COVID-19

Pathogenesis is closely linked with the clinical presentation of the COVID-19 disease caused by this virus.
Reports have described diverse symptom profiles associated with COVID-19, with a great deal of variability both within and between institutions and regions.

> A large study ffrom Wuhan, China conducted early in the pandemic identified fever and cough as the two most common symptoms that patients reported at hospital admission [@doi:10.1056/NEJMoa2002032], while a retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].

Reviewer 3: ```suggestion
A large study from Wuhan, China conducted early in the pandemic identified fever and cough as the two most common symptoms that patients reported at hospital admission [@doi:10.1056/NEJMoa2002032], while a retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
```

This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, suggesting that the virus preferentially targets cells located in the lower respiratory tract.
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across individuals.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in 20.4% of patients upon presentation, with fever being present in 30.7%, while another study [51] reported cough (79.4%), fever (77.1%), and dyspnea (56.5%) as the as the most common presenting symptoms; both of these studies considered only hospitalized patients.

> The variability of both which symptoms present and their severity has presented challenges for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.

Author: ```suggestion
```
Thanks @agitter! This was removed in #796.
@ajlee21, I checked carefully and I am pretty confident that all of the corrections to duplications from #772 are fixed here! It also looks like a unique sentence was deleted and it's added back in below (line 142)

Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, omics technologies offer the opportunity for discovery-driven analysis of biological changes associated with SARS-CoV-2 infection.
For example, previous studies suggest that infection by coronaviruses, such as SARS-CoV-1 and MERS-CoV, as well as other viruses, is associated with the upregulation of ACE2.
In several preliminary assays and an analysis of microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].

> While it is still unclear whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.

Reviewer: This it should be `to characterize` here

This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
While it is still unclear whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.

#### Transcriptomics

In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines.
The first study [@doi:10.1016/j.cell.2020.04.026] compared transcriptional responses to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).

> They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).

Reviewer 3: "They" now refers to "The first study".  That doesn't read naturally to me.

Author: ```suggestion
The responses of three human cell lines were analyzed: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
```
Passive voice to the rescue!

For example, previous studies suggest that infection by coronaviruses, such as SARS-CoV-1 and MERS-CoV, as well as other viruses, is associated with the upregulation of ACE2.
In several preliminary assays and an analysis of microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
While it is still unclear whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.

> #### Transcriptomics

Reviewer 4: I agree. But during that time (last ~ April - May) these were the only 2 "major" published studies. I would like to expand this to scRNA-seq studies. However, we decided to do that in the next update.

In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines.
The first study [@doi:10.1016/j.cell.2020.04.026] compared transcriptional responses to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, these cells were supplemented with adenovirus (AdV)-based vectors expressing either mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes from an animal model (ferret), and lung samples from two COVID-19 patients.

> Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.

Reviewer 4: Actually I specifically added this sentence after some confusion was raised (along the same lines as you wrote :) ). DE analysis was performed  infected cells vs mock control cells.

In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique pro-inflammatory cytokine signature associated with SARS-CoV-2 was present in cells exposed to both high and low doses of the virus, with the cytokines IL-6 and IL1RA uniquely elevated in response to SARS-CoV-2 relative to other viruses.
However, the A549-ACE2 cells showed significant IFN-I or IFN-III expression when exposed to high, but not low, doses of SARS-CoV-2.
This finding suggests that IFN induction is dependent on the extent of exposure.
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, chemokine signaling was significantly enriched, but there was no significant induction of IFN-I or IFN-III.
Together, these results suggest that SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated gene response, in contrast to other viruses.

> However, in ACE2-expressing A549 cells, this state could be overcome by using a 10-fold increase in SARS-CoV-2 exposure.

Reviewer 4: I am not sure if I understood your question?

This finding suggests that the SARS-CoV-2 interferon antagonist is insufficient for large doses of the virus [@doi:10.1016/j.cell.2020.04.026].

> This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 *ORF3b* gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene.

Reviewer 4: Good point. May be we can extend Fig 1 for accessory proteins at 3 prime. Something like Figure 3 here https://inflammregen.biomedcentral.com/articles/10.1186/s41232-020-00126-7#Fig3
![image](https://user-images.githubusercontent.com/51865783/105158640-3628c280-5b34-11eb-8532-f18335890671.png)
![image](https://user-images.githubusercontent.com/51865783/105159071-b0594700-5b34-11eb-9314-bd2890c4865c.png)

> This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 *ORF3b* gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene.

Reviewer 3: Yes, I believe the italics render correctly.  There needs to be whitespace before and after the italicized word.

I also made this suggested change in my proofreading.

> This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 *ORF3b* gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene.

Reviewer 4: @rando2 Could we please update citation number 78 and 150? The 2 papers were published in a journal (peer-reviewed). Currently, they are cited for bioarxiv.

The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 and show rapid viral replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935].
This discrepancy raises the concern that _in vitro_ models may vary in their similarity to the human response, underscoring the importance of follow-up studies in additional models.

> #### Proteomics 

Author: ```suggestion
#### Proteomics
```

Consequently, proteomic analyses of SARS-CoV-1 might also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].

> Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.

Author: @SiminaB says:

> Ideally we should cite these

Author: ```suggestion
Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt [@doi:10.1093/nar/gky1049], NCBI Genome Database [@url:https://www.ncbi.nlm.nih.gov/genome/], The Immune Epitope Database and Analysis Resource (IEDB) [@url:www.iedb.org; @doi:10.1093/nar/gky1006], and The Virus Pathogen Resource (ViPR) [@url:http://www.viprbrc.org/] will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
```

Differences in how the S proteins of the two viruses interact with the hACE2 receptor could also partially account for the increased transmissibility of SARS-CoV-2.
Recent studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity than the SARS-CoV-1 S protein does [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV-1 [@doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV-1 S protein, eight are conserved in SARS-CoV-2, and the remaining six are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].

> Recent crystal structures have shown that the receptor-binding domain (RBD) of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].

Author: This change (adding definition of RBD) was made in #783 -- I believe this is correct!

Nevertheless, despite the high structural homology between the SARS-CoV-2 RBD and that of SARS-CoV-1, monoclonal antibodies targeting SARS-CoV-1 RBD failed to bind to SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].

> Promisingly, though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry _in vitro_, albeit with lower efficiency than it inhibited SARS-CoV-1 [@doi:10.1016/j.cell.2020.02.052].

Reviewer: Personally think this was easier to read when split into 2 sentences.

### Viral Evolution and Virulence

> Like that of SARS-CoV-1, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein, S, and human ACE2 (hACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20]. 

Author: @SiminaB wrote:

> We call it ACE2 earlier and this generally is focused on humans, so do we need to have it as hACE2 here?

@ajlee21 had a similar question on #784. My suspicion is that we should use hACE2 when talking about human/clinical studies and ACE2 when talking about broader studies (_in vitro_ etc.) but I am not totally sure what the distinction would be.

Author: ```suggestion
Like that of SARS-CoV-1, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein, S, and human ACE2 (hACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
```

Differences in how the S proteins of the two viruses interact with the hACE2 receptor could also partially account for the increased transmissibility of SARS-CoV-2.
Recent studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity than the SARS-CoV-1 S protein does [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV-1 [@doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV-1 S protein, eight are conserved in SARS-CoV-2, and the remaining six are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].

> Recent crystal structures have shown that the RNA-binding domain (RBD) of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].

Author: ```suggestion
Recent crystal structures have shown that the receptor binding domain (RBD) of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
```

Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 could represent a promising therapeutic approach.
Nevertheless, despite the high structural homology between the SARS-CoV-2 RBD and that of SARS-CoV, monoclonal antibodies targeting SARS-CoV-1 RBD failed to bind to SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].

> Promisingly, though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry _in vitro_, albeit with lower efficiency than it inhibited SARS-CoV-1 [@doi:10.1016/j.cell.2020.02.052].

Author: @SiminaB says:

> I know this isn’t the Therapeutics paper, but I’m not sure how much to emphasize the convalescent sera here. I do know that studies of convalescent plasma haven’t shown a benefit.

Once a human host is infected with a virus, person-to-person viral transmission of a virus can occur through several possible mechanisms.
The primary mechanisms associated with respiratory viruses are contact, droplet, and aerosol transmission [@doi:10.1097/QCO.0000000000000563].

> Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface [@pubmed:23771256]. 

Author: @SiminaB says:

> What’s “direct contact” here? Doesn’t it still have to come from droplets or aerosols?

> Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface [@pubmed:23771256]. 

Author: ```suggestion
Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface [@pubmed:23771256].
```

This latter mode of transmission is also called fomite transmission [@url:https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions].
Viral particles can enter the body if they then come in contact with the oral, nasal, eye, or other mucus membranes [@pubmed:23771256].
Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles [@pubmed:23771256].

> Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@pubmed:23771256]. 

Author: @SiminaB says:

> How is this different from fomite transmission (described above)?

Author: ```suggestion
Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@pubmed:23771256].
```

However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458], and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].
These mechanisms may differ in their relevance to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.

#### Symptoms and Viral Spread

Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness.
Symptoms associated with reported cases of COVID-19 range from mild to severe [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html], but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness [@doi:10.1007/s11427-020-1661-4].

> The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE], and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms (95% CI, -0.9 - 0.9 days) [@doi:10.1038/s41591-020-0869-5].

Author: @SiminaB says:

> I think this is probably the median length – please check. I was looking into this a couple of weeks ago and there is a distribution of incubation times that has a long tail.

Reviewer: Perhaps a related question. I think I asked about this already so I apologize for asking again. So The way I'm thinking about this is a little confusing. Say your day of symptom onset = 0. Then based on this description, your exposure day = - 5, but your viral shedding can start at day -12. How can you have viral shedding start before your exposure? Though perhaps I'm misunderstanding what they mean by exposure.

Author: @ajlee21 I think it's easier to think about it in the other direction -- if you're exposed to a virus on day X, you are probably going to start showing symptoms around day X+5 and shedding the virus on day X+4 to X+6. However, it's possibly you get exposed on day X, don't show symptoms for a long time, and start viral shedding 12 days before you start showing symptoms. Simina pointing out the long tail really helped me understand why these time periods seem so weirdly long!!!

As these trends became apparent, concerns arose due to the potential for individuals who did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
Recovered individuals may also be able to transmit the virus after their symptoms cease.

> Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].

Author: @SiminaB says:

> Is this different from the period in which they can shed the virus?

The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical of atypical symptoms [@doi:10.1056/NEJMoa2008457].
Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887].

> However, viral shedding is not necessarily indicative of contagiousness. 

Author: @SiminaB says:

> Seems like the answer to the above question is: No. However, we may want to mention this earlier, as it can be confusing. Note that we also use both the words “contagious” and “communicable.” – I think they’re synonyms.

> However, viral shedding is not necessarily indicative of contagiousness. 

Author: ```suggestion
However, viral shedding is not necessarily indicative of contagiousness.
```

Recovered individuals may also be able to transmit the virus after their symptoms cease.
Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].
In an early study, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, in later studies, viral shedding was reported for up to 28 days following symptom onset [@doi:10.1038/s41586-020-2196-x] and for up to 24 days from first positive PCR test, with a median of 12 days [@doi:10.1001/jama.2020.3204].

> The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].

Reviewer 3: The CVS acronym was removed in #790.  Will have to clean that up in a merge commit.

Author: I'll keep an eye out, thanks for pointing this out!

For example, in later studies, viral shedding was reported for up to 28 days following symptom onset [@doi:10.1038/s41586-020-2196-x] and for up to 24 days from first positive PCR test, with a median of 12 days [@doi:10.1001/jama.2020.3204].
The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical of atypical symptoms [@doi:10.1056/NEJMoa2008457].

> Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887]).

Reviewer 3: The close parenthesis is needed.

> Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887].

Author: ```suggestion
Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887]).
```

The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus raises significant concerns.
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were considered to be caused by contact with asymptomatic individuals [@doi:10.7554/eLife.57149].
An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
The argument was thus made that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.

> #### Estimating the Fatality Rate

Author: @SiminaB says:

> Wondering if it would help break up the text a bit to have the IFR and CFR mathematical definitions on separate lines:
> CFR = (number of COVID-19 related deaths)/(number of confirmed or suspected cases)
> IFR = (number of COVID-19 related deaths)/(number of COVID-19 cases)

> This would also emphasize that the numerator is the same.

Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One commonly reported metric is the case fatality rate (CFR), which compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.

> Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780].

Author: @SiminaB says:

> Is this just in the Ioannidis paperi or in other sources as well? I tend to always see estimates between 0.6-1%.

Author: ```suggestion
IFR varies regionally, with some locations observed to have IFRs as low as 0.17% while others are as high as 1.7% [@doi:10.1016/j.ijid.2020.09.1464].
Estimates of CFR at the national and continental level and IFR at the content level is maintained by the Centre for Evidence-Based Medicine [@url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/].
Several meta-analyses have sought to estimate IFR at the global scale.
These estimates have varied; one peer-reviewed study aggregated data from 24 other studies and estimated IFR at 0.68% (95% CI 0.53%–0.82%), but a preprint that aggregated data from 139 countries calculated a global IFR of 1.04% (95% CI 0.77%-1.38%) when false negatives were considered in the model [@doi:10.1101/2020.05.11.20098780] and  [@doi:10.1016/j.ijid.2020.09.1464]. 
```

This was also supported by a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].

> All of these estimates note that IFR varies widely around the world.

Author: @SiminaB says:

> They also vary a lot with age, sex, comorbities. I think we have this discussion in one of our other planned papers, so we may want to cite that. We may also want to say that estimated CFR varies a lot more by country, especially in the early stages of the pandemic when there’s less testing and health systems are overburdened eg https://ourworldindata.org/mortality-risk-covid

Author: ```suggestion
All of these studies note that IFR varies widely by location, and it is also expected to vary with demographic and health-related variables such as age, sex, prevalence of comorbidities, and access to healthcare and testing [@url:https://ourworldindata.org/covid-mortality-risk].
```

#### Estimating the Fatality Rate

Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One commonly reported metric is the case fatality rate (CFR), which compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780].

> This was also supported by a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].

Author: I had learned in high school that "this" can't be a noun on its own -- there either needs to be a predicate noun ("This is an excellent study") or a noun directly after it (e.g., "This finding was also supported...").  However, I cannot for the life of me find/remember the name of this rule!!! That's why I originally had "This estimate" and had stuck a lot of nouns in while giving people feedback, but now I'm worrying I made up this rule...

Reviewer 3: I've heard similar guidelines because it can lead to an [unclear antecedent](https://iconlogic.blogs.com/weblog/2010/11/grammar-workshop-can-i-start-a-sentence-with-this.html).

Author: ```suggestion
A similar prevalence estimate was identified through a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].
Examination of serosurvey-based estimates of IFR identified convergence on a global IFR  estimate of 0.60% (95% CI 0.42%–0.77%) [@doi:10.1016/j.ijid.2020.09.1464].
```

### Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
Accurate estimates are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174].
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model.
In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174].

> A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996]. 

Author: @SiminaB says:

> I’m a bit unclear on this. It only takes 1 person to be infected. I know that if Rt < 1 eventually the infectious dies down, but I’m confused on this issue of “invading a susceptible population”.

Author: I'm tagging @LucyMcGowan, @alavendelm, and @rdvelazquez for help on this one! I think I understand the idea being communicated but I'm not sure exactly how to adjust the phrasing.

> A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996]. 

Author: ```suggestion
A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
```

Another estimate based on a cruise ship where an outbreak occurred predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SEIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].
Using the same model as for the French population, a study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~ was estimated to be 2.79.

Inference of the effective reproduction number can provide insight into how populations respond to an infection and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in January 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (January 23, 2020), to 1.05 one week after.

> Using their model, the authors also estimated the probability of new outbreaks occurring: the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%. 

Author: @SiminaB says:

> Which would be what?

Author: ```suggestion
Using their model, the authors also estimated the probability of new outbreaks occurring.
Assuming individual-level variation in transmission comparable to that of MERS or SARS, the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, and assuming variation like that of SARS and transmission patterns like those observed for COVID-19 in Wuhan, the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
```

> Using their model, the authors also estimated the probability of new outbreaks occurring: the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%. 

Author: ```suggestion
Using their model, the authors also estimated the probability of new outbreaks occurring: the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
```

Another estimate based on a cruise ship where an outbreak occurred predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SEIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].
Using the same model as for the French population, a study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].

> From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~ was estimated to be 2.79.

Author: @SiminaB says:

> I assume this is across studies?

Author: ```suggestion
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] the median R~0~ was estimated to be 2.79 (IQR 1.16) based on twelve studies published between  January 1 and February 7, 2020.
```
